README.md

```diff
@@ -1,12 +1,9 @@
-Description
------------
+# stm32-enc28j60
 
-An implementation of ENC28J60 driver for STM32 MCU. Tested only on sending UDP packets and only on MCUs STM32F091xC, STM32F030x6 and STM32F103xB. If you need to use another STM32 MCU then don't forget to edit file `inc/stm32includes.h`.
+STM32 ENC28J60 ethernet driver, ported for CMake using [stm32-base](https://github.com/DVALight/stm32-base)
 
-Working examples
-----------------
+## CMake
 
-* [dc-thermal-logger](https://devel.mephi.ru/dyokunev/dc-thermal-logger/src/master/collector/firmware/Src/main.c)
-* [CardLogger](https://devel.mephi.ru/AMGladtsin/STM32_Devel/src/master/SOFT/CardLogger/Src/main.c)
-* [enc28j60-helloWorld](https://devel.mephi.ru/dyokunev/nucleo-f091rc-rs232-enc28j60/src/master/Src/main.c)
+Builds as library that can be linked using CMake in other STM32 projects.
 
+Fully configurable, use variables defined in CMakeLists.txt to enable, disable and configure internet modules
```

Comment:
Could you give a link on an example how to use this library (since the old examples are obsolete and removed)?

Reply:
Examples that you mentioned in README are good and works fine with my PR, but unfortunately they are on russian git site that I cant access even with Tor. It would be awesome if you reupload them on github so I can add in PR's README

Reply:
Ah, I see. I guess goddamn RKN or something got insane. Cloned repositories to:
* https://github.com/mephi-ut/dc-thermal-logger
* https://github.com/mephi-ut/STM32_Devel
* https://github.com/mephi-ut/nucleo-f091rc-rs232-enc28j60

Reply:
Thanks. I will add examples to PR's readme